src/api/queries/videos.graphql

```diff
@@ -102,6 +102,20 @@ query GetBatchedVideoViews($videoIdList: [ID!]!) {
   }
 }
 
+query GetMostViewedVideos($viewedWithinDays: Int!, $limit: Int) {
+  mostViewedVideos(period: $viewedWithinDays, limit: $limit) {
+    id
+    views
+  }
+}
+
+query GetMostViewedChannels($viewedWithinDays: Int!, $limit: Int) {
+  mostViewedChannels(period: $viewedWithinDays, limit: $limit) {
+    id
+    views
+  }
+}
```

Comment:
This likely should go to `channels.graphql` right?

Reply:
🤦